pilot/pkg/config/clusterregistry/conversion_test.go

```diff
@@ -15,26 +15,34 @@
 package clusterregistry
```

Comment:
Still going through the patch but the size of this change to conversion_test is feeling like too big of a refactor to fit the description of moving from file-based to configmap & secret.  I was thinking you'd only need to change getting the test code that is dependent on storing config in files and use of conversion.go APIs dependent on files.  This is looking a lot more extensive than that.

Reply:
right and the reason is conversion.go mostly dealt with getting config from file based storage  and checking content. Since all file related things are gone, then there is no any sense to test them, right? conversion_test.go was refactored for tests which make sense for new approach.